internal/pkg/errors/error_message.go

```diff
@@ -209,7 +209,7 @@ const (
 	KsqlDBTerminateClusterErrorMsg = `failed to terminate ksqlDB cluster "%s" due to "%s"`
 
 	// local commands
-	ConfluentLocalStartedErrorMsg    = "Confluent local container has been created"
+	ConfluentLocalStartedErrorMsg    = "Confluent local container has already been created"
```

Comment:
Nit: I assume the product is called "Confluent Local" (both words uppercase)
```suggestion
	ConfluentLocalStartedErrorMsg    = "Confluent Local container has already been created"
```